content/learning-paths/cross-platform/simd-on-rust/conclusion.md

```diff
@@ -6,22 +6,11 @@ weight: 7
 layout: learningpathall
 ---
 
-You have seen a few aspects of writing SIMD code on Arm with Rust. In the end which is better, C or Rust?
+You have seen a few examples of writing SIMD code on Arm with Rust. 
 
-Performance wise there is no difference really, as you have seen. Rust is perfectly capable of generating the same assembly code as C in most cases.
+Performance wise, between C and Rust there is no difference really. Rust is perfectly capable of generating the same assembly code as C in most cases.
 
-If you want to program optimal SIMD code using the Arm ASIMD/Neon intrinsics, then `std::arch` is the obvious choice.
+If you want to program optimal SIMD code using the Arm ASIMD/Neon intrinsics, then `std::arch` is the most obvious choice.
 
-If however, your approach needs to be as portable as possible and you don't want to spend time providing multiple implementations for each architecture, then `std::simd` is also starting to become a very viable alternative, even though it's not part of the stable compiler yet.
+If however, your approach needs to be as portable as possible and you don't want to spend time providing multiple implementations for each architecture, then `std::simd` is a very viable alternative, even though it's not part of the stable compiler yet.
 
-So in the end it is mostly about your personal preference, Rust is a modern language and provides many new features that are hard to be ignored even by the most hardcore C and C++ developers. It is possible to write safe code with C and C++, but it requires more effort and more tools, whilst Rust provides that by default.
-
-### What about SVE, SVE2 or even SME?
```

Comment:
This section was requested to be added by Arm Rust engineers, and the link to the RFC was given to be added. I would still include it, perhaps you would like to rephrase it if it doesn't meet the guidelines.